python/cudf_polars/cudf_polars/dsl/expressions/aggregation.py

```diff
@@ -124,6 +77,46 @@ def __init__(
         "linear": plc.types.Interpolation.LINEAR,
     }
 
+    def _fill_request(self):
```

Comment:
Maybe we can just define `request` as a property (or cached property), and move this same logic there?

```python
@property
def request(self):
   ...
``

Reply:
Good idea. Done in https://github.com/rapidsai/cudf/pull/17449/commits/faf42d06bc3a465f7d93e1dbd0621e5953151e69 .